prov/shm/src/smr_progress.c

```diff
@@ -200,6 +209,78 @@ static int smr_progress_iov(struct smr_cmd *cmd, struct iovec *iov,
 	return -ret;
 }
 
+static int smr_mmap_iov_copy_out(struct smr_ep *ep, struct smr_cmd *cmd,
+				 struct iovec *iov, size_t iov_count,
+				 size_t *total_len)
+{
+	char shm_name[NAME_MAX];
+	char peer_name[NAME_MAX];
+	void *mapped_ptr;
+	int peer_id, fd, ret, num;
+
+	peer_id = (int) cmd->msg.hdr.addr;
+
+	sprintf(peer_name, "%s", ep->region->map->peers[peer_id].peer.name);
+	num = sprintf(shm_name, "%s_%s_%ld", peer_name, ep->name,
+		      cmd->msg.hdr.msg_id);
+	if (num < 0) {
+		FI_WARN(&smr_prov, FI_LOG_AV, "generating shm file name failed\n");
+		return errno;
```

Comment:
Negative errno would be consistent with the rest of the code base.

Reply:
Done.